app/_src/mesh/sbom.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Software Bill of Materials
+toc: false
+---
+
+A software bill of materials (SBOM) is an inventory of all software components (proprietary and open source), open source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
+
+Starting in {{site.mesh_product_name}} 2.7.4, we are generating SBOMs for {{site.mesh_product_name}} and docker container images.
+
+1. [Download security assets](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-release/packages/?q=name%3Asecurity-assets+version%3A{{page.kong_latest.version}}) for the latest version of {{site.mesh_product_name}}
```

Comment:
Same should we just link to the direct download link?

Reply:
done.